odoo/tools/misc.py

```diff
@@ -1148,6 +1148,8 @@ def difference_update(self, elems):
     def __repr__(self):
         return f'{type(self).__name__}({list(self)!r})'
 
+    intersection = property(lambda self: self.__and__)
```

Comment:
Sry, I told you that I was ok. But I rechecked this morning, and it actually not completely correct.
`set.Intersection` can manage multiple argument and not only 2. 

```suggestion
    def intersection(self, *others):
        current = dict(self._map)
        for other in others:
            current &= other
        return OrderedSet(current)
```